hacknight1/hacknight1.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,242 @@
+// !$*UTF8*$!
```

Comment:
Can you please add a .gitignore file for this? The only thing that you commit should be your main.cpp files